classes/mail_wrapper.py

```diff
@@ -25,6 +25,5 @@ def get_sender(self):
 
```

Comment:
Usually commit messages are don in an imperative format, take a look in https://chris.beams.io/posts/git-commit/